engine/glfw/lib/android/android_window.c

```diff
@@ -340,13 +354,16 @@ void glfwAndroidFlushEvents()
             // The first time, the create_gl_surface() is called from the _glfwPlatformOpenWindow function
             if (_glfwWin.opened && _glfwWinAndroid.display != EGL_NO_DISPLAY && _glfwWinAndroid.surface == EGL_NO_SURFACE)
             {
-                create_gl_surface(&_glfwWinAndroid);
-                make_current(&_glfwWinAndroid);
-                update_width_height_info(&_glfwWin, &_glfwWinAndroid, 1);
-
-                computeIconifiedState();
+                CreateGLSurface();
             }
             break;
+
+
+        case APP_CMD_GAINED_FOCUS:
+            // If we failed to create the window in APP_CMD_INIT_WINDOW, let's try again
+            if (_glfwWinAndroid.surface == EGL_NO_SURFACE) {
+                CreateGLSurface();
+            }
```

Comment:
The fix, is to handle the first failed creation gracefully, and here try to create it again, just before we present it to the user.